browser/components/WYSIWYGEditor.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import ProseMirror from 'react-prosemirror'
+import style from './WYSIWYGEditor.styl'
+import CSSModules from 'browser/lib/CSSModules'
+
+import 'prosemirror/dist/inputrules/autoinput'
+import 'prosemirror/dist/markdown'
+
+function buildStyle (fontSize) {
+  return {
+    fontSize: `${fontSize}px`
+  }
+}
+
+class WYSIWYGEditor extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      value: props.value,
+      config: props.config
```

Comment:
You don't need to set `config` state.
This component won't change `config`

Reply:
fix by c53542e 😃